src/components/projects/CategoryInput.tsx

```diff
@@ -59,12 +61,25 @@ export const CategoryInput: React.FC<{
   const options = useMemo(() => {
     const options: OptionType[] = []
     sectionCategories.map((cat) => {
+      // check if the category is part of the backmatter section, and its already present in the document
+      let isDisabled = false
+      if (
+        cat.groupIDs &&
+        isBackMatterSection(cat.groupIDs[0]) &&
+        document.querySelector(`[data-category="${cat._id}"]`)
```

Comment:
Please check against the prosemirror doc instead. This will also be safer, since in case we'd make changes to the section attributes, it'd be reflected on the type and will not build.